Added airgapped instructions to the docs (#5108)
aws/eks-anywhere: docs/content/en/docs/getting-started/install/_index.md

eksctl anywhere version
```

## Prepare for airgapped deployments (optional)

When creating an EKS Anywhere cluster, there may be times where you need to do so in an airgapped
environment.
In this type of environment, cluster nodes are connected to the Admin Machine, but not to the
internet.
In order to download images and artifacts, however, the Admin machine needs to be temporarily
connected to the internet.

An airgapped environment is especially important if you require the most secure networks.
EKS Anywhere supports airgapped installation for creating clusters using a registry mirror.
For airgapped installation to work, the Admin machine must have:

* Temporary access to the internet to download images and artifacts

> * Ample space (50 GB or more) to store artifacts locally

Reviewer: We had a 50Gb admin machine in vSphere, which couldn't store all the artifacts. We increased the disk size to 80. So I think we should recommend 80Gb here.

Author: Fixed.

To create a cluster in an airgapped environment, perform the following:

1. Download the artifacts and images that will be used by the cluster nodes to the Admin machine using the following command:
   ```bash
   eksctl anywhere download artifacts
   ```
   A compressed file `eks-anywhere-downloads.tar.gz` will be downloaded.

1. To decompress this file, use the following command:
   ```bash
   tar -xvf eks-anywhere-downloads.tar.gz
   ```
   This will create an eks-anywhere-downloads folder that we’ll be using later.

> 1. In order for the next command to run smoothly, ensure that Docker has been preinstalled. Then run the following:

Reviewer 2: ```suggestion
1. In order for the next command to run smoothly, ensure that Docker has been pre-installed and is running. Then run the following:
```

Author: Done.

   ```bash
   eksctl anywhere download images -o images.tar
   ```

   **For the remaining steps, the Admin machine no longer needs to be connected to the internet or the bastion host.**

> 1. Next, you will need to set up a local registry mirror to host the downloaded EKS Anywhere contents. In order to set one up, refer to [Registry Mirror configuration.]({{< relref "/docs/reference/clusterspec/optional/registrymirror.md" >}})

Reviewer 2: nit: more precise?
```suggestion
1. Next, you will need to set up a local registry mirror to host the downloaded EKS Anywhere images. In order to set one up, refer to [Registry Mirror configuration.]({{< relref "/docs/reference/clusterspec/optional/registrymirror.md" >}})
```

Author: Done.

1. Now that you’ve configured your local registry mirror, you will need to import images to the local registry mirror using the following command (be sure to replace <registryUrl> with the url of the local registry mirror you created in step 4):
   ```bash
   eksctl anywhere import images -i images.tar -r <registryUrl> \

>       -- bundles ./anywhere-downloads/bundle-release.yaml

Reviewer 2: ```suggestion
      -- bundles ./eks-anywhere-downloads/bundle-release.yaml
```

Author: Done.

   ```
You are now ready to deploy a cluster by following instructions to [Create local cluster]({{< relref "/docs/getting-started/local-environment/" >}}) or  [Create production cluster.]({{< relref "/docs/getting-started/production-environment/" >}}) See text below for specific provider instructions.

### For Bare Metal (Tinkerbell)

> You will need to have hookOS and its OS artifacts downloaded and served locally from an http file server.

Reviewer 2: ```suggestion
You will need to have hookOS and its OS artifacts downloaded and served locally from an HTTP file server.
```

Author: Done.
